otelcollector/opentelemetry-collector-builder/collector-config-template.yml

```diff
@@ -36,7 +36,6 @@ processors:
 receivers:
     prometheus:
       api_server:
-        enabled: true
```

Comment:
Could we double check if this is intentional?

Reply:
Yes thanks this is, the new optionalconfig type prohibits enabled and now if it's in the config, that counts as enabled